src/project/FileTreeView.js

```diff
@@ -478,7 +478,42 @@ define(function (require, exports, module) {
         render: function () {
             var fullname = this.props.name,
                 extension = LanguageManager.getCompoundFileExtension(fullname),
-                name = _getName(fullname, extension);
+                name = _getName(fullname, extension),
+                fileType = "default";
+
+            switch (extension.toLowerCase()) {
+              case 'htmls':
```

Comment:
I see. I added the '.' because of this code.
https://github.com/mozilla/brackets/blob/master/src/filesystem/impls/filer/lib/content.js#L36-L106
: )